content/en/serverless/configuration/_index.md

```diff
@@ -458,7 +458,7 @@ If you are using a runtime or custom logger that isn't supported, follow these s
 
 ## Link errors to your source code
 
-<div class="alert alert-info">This feature is supported for Go and Java.</div>
+<div class="alert alert-info">This feature is currently supported for Go, Java, Python and Node.js.</div>
```

Comment:
Small suggestion to be consistent with [SCI docs](https://docs.datadoghq.com/integrations/guide/source-code-integration/?tab=github) and remove any ambiguity regarding typescript
```suggestion
<div class="alert alert-info">This feature is currently supported for Go, Java, Python and JavaScript.</div>
```